src/features/create-token/TokenSummaryField/TokenSummaryField.tsx

```diff
@@ -0,0 +1,23 @@
+//- React Imports
+import { FC } from 'react';
+
+//- Style Imports
+import styles from './TokenSummaryField.module.scss';
+import classnames from 'classnames';
+
+interface TokenSummaryFieldProps {
+	className?: string;
+	label: string;
+	value: string | number;
+}
+
+export const TokenSummaryField: FC<TokenSummaryFieldProps> = ({
+	className = '',
```

Comment:
Shouldn't need to set `''` as default - I believe `classnames` can take `undefined`.